src/plugin.js

```diff
@@ -87,6 +89,29 @@ const initPlugin = function(player, options) {
   // Make sure we flesh out initially-provided errors.
   updateErrors();
 
+  const iOSPlayback = function() {
```

Comment:
Rather than having extra device detection, videojs provides a helper for that already:
```js
videojs.browser.IS_IOS
```